src/pages/html/tutorials/how-to-create-an-html-button-that-acts-like-a-link/index.md

```diff
@@ -1,15 +1,47 @@
 ---
 title: How to Create an HTML Button That Acts Like a Link
 ---
-## How to Create an HTML Button That Acts Like a Link
+This guide will demonstrate "How to Create an HTML Button That Acts Like a Link".
+Programmers and Web Developers use buttons for numerous different occasions. Some Buttons are for submit forms such as emails, newsletters, or polls and etc. While others are to open links or bring up pages.
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/html/tutorials/how-to-create-an-html-button-that-acts-like-a-link/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Today you will create your very own social link button! 
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+## Button Tag
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+This is the first step and it is the step that defines to the computer
+what you want to see. Lets Start! Type the following as shown below.
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+`<button>`
+  
+## Best Practice
 
+It is good to get in the habit of closing your tags first before writing code. To avoid any syntax errors and to insure your code runs smoothly. It may seem unimportant with the line of code demonstrated in this guide.
+
+However there will be times when you will have a numerous lines. It is very simple to forget to close your`<tag>`s or `<div>`s resulting in bad or not working code. 
+
+So lets close the button tag, like so...
+
+`<button></button>`
+
+## Anchor Tag
+
+An anchor tag is one that you can use to bring up a local (on your computer) or network (internet) file or page. For this demonstration you will need to put an anchor tag within your button tag.
+  
+`<button><a href="Put_Your_Facebook_link_here_"</a>Add Text Here Such As "Twitter"</button>`
+
+## Test Your Code
+
+Before you test your new social link button. Look over your code for errors such as an unclosed `<tag>` or `<div>` and or misspelled words. 
```

Comment:
Replace "Look over your code for errors such as an unclosed `<tag>` or `<div>` and or misspelled words. " with "Look at your code for errors such as an unclosed `<element>`, `<div>`, or misspelled words. "